reSnap.sh

```diff
@@ -11,6 +11,7 @@ fi
 # default values
 ip="${REMARKABLE_IP:-10.11.99.1}"
 output_file="$tmp_dir/snapshot_$(date +%F_%H-%M-%S).png"
+ssh_key_file=""
```

Comment:
Id like to have this also configurable via environment variables, similarly to the ip.

Maybe the name of the var being `REMARKABLE_SSH_KEY_FILE`